### PR TITLE
Notification wording edits

### PR DESCRIPTION
**Pull request by monthrie**
mostly just putting the item name in single quotes and adding a full stop here and there

### CODE REVIEW

File: react-ui/public/service.js [hunk 1]
```diff
@@ -656,8 +658,8 @@ function processItemSentClicked(entity) {
     updateListing(id, { "status": "completed", 'notification': true });
     getListingById(id, function (listing) {
         //MDS.log(`Listing found: ${JSON.stringify(listing)}`);
-        notification(`Your purchase ${listing.title} has been sent by @${listing.created_by_name} and is now in transit.`);
-        var message = `Your purchase ${listing.title} has been sent by @${listing.created_by_name} and is now in transit.`
+        notification(`Your purchase '${listing.title}' has been sent by @${listing.created_by_name} and is now in transit.`);
+        var message = `Your purchase ''${listing.title}' has been sent by @${listing.created_by_name} and is now in transit.`
```
Comment (dgowell) [inline react-ui/public/service.js]: this looks like there are one double quotes and one single quote

File: react-ui/public/service.js [hunk 2]
```diff
@@ -712,12 +714,13 @@ function processCollectionRequest(entity) {
     updateListing(id, {
         'buyer_message': entity.message,
         'status': 'ongoing',
+        'notification': true,
```
Comment (dgowell) [inline react-ui/public/service.js]: did you add this back?

File: react-ui/public/service.js [hunk 3]
```diff
@@ -1087,6 +1090,7 @@ function createListingTable(callback) {
             "purchase_code" varchar(30),
             "pendinguid" varchar(34) default null,
             "coin_id" varchar(80),
+            "notification" boolean default false,
```
Comment (dgowell) [inline react-ui/public/service.js]: i removed this as it's not needed

**Review by dgowell — CHANGES_REQUESTED** [review]
are you able to review the comments ?

File: react-ui/public/service.js (hunk 1 above)
Comment (monthrie) [reply react-ui/public/service.js]: hahaha. i agree.

**Review by dgowell — APPROVED** [review]

**Pull request merged**